bindsnet/network/nodes.py

```diff
@@ -398,6 +405,13 @@ def forward(self, x: torch.Tensor) -> None:
         self.refrac_count.masked_fill_(self.s, self.refrac)
         self.v.masked_fill_(self.s, self.reset)
 
+        # Check for lower boundself.
+        lb_check = self.v < self.lbound
+
+        # voltage clipping to lowerbound
+        self.v.masked_fill_(lb_check, self.lbound)
```

Comment:
The above two lines can be combined into one:

`self.v.masked_fill_(self.v < self.lbound, self.lbound)`